### PR TITLE
Adding an updated_at field to the ServiceStatuses table.

### PR DESCRIPTION
**Pull request by TimSimpsonR**

### CODE REVIEW

**Comment by cp16net** [comment]
+1

**Comment by TimSimpsonR** [comment]
@sacharya We have to set this each time the agent updates the field, or otherwise it'll be difficult to impossible to determine if the agent has died without logging into the box.

File: reddwarf/db/sqlalchemy/migrate_repo/versions/003_service_statuses.py [hunk 1]
```diff
@@ -36,7 +36,8 @@
     Column('id', String(36), primary_key=True, nullable=False),
     Column('instance_id', String(36), nullable=False),
     Column('status_id', Integer(), nullable=False),
-    Column('status_description', String(64), nullable=False))
+    Column('status_description', String(64), nullable=False),
+    Column('updated_at:', DateTime()))
```
Comment (sacharya) [inline reddwarf/db/sqlalchemy/migrate_repo/versions/003_service_statuses.py]: updated: ?

Comment (sacharya) [inline reddwarf/db/sqlalchemy/migrate_repo/versions/003_service_statuses.py]: I mean the colon :

Comment (hub-cap) [inline reddwarf/db/sqlalchemy/migrate_repo/versions/003_service_statuses.py]: u updated your colon?

Comment (ed-) [inline reddwarf/db/sqlalchemy/migrate_repo/versions/003_service_statuses.py]: Yeah what's up with that

Comment (cp16net) [inline reddwarf/db/sqlalchemy/migrate_repo/versions/003_service_statuses.py]: haha good catch @sacharya

**Comment by TimSimpsonR** [comment]
@sacharya Wow, sorry I missed that you were pointing out a typo. Thanks for catching that!

After talking with Nirmal and Zach a bit we decided to add a heart beats table as well, even though we'd like to get away from the agents talking to the MySQL db directly...

**Comment by hub-cap** [comment]
+1

**Comment by sacharya** [comment]
+1

**Comment by ed-** [comment]
+1

**Comment by rnirmal** [comment]
+1

**Pull request merged**